rules/aws/privilege_escalation_sts_getsessiontoken_abuse.toml

```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2021/05/17"
+maturity = "development"
+updated_date = "2021/05/17"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies the suspicious use of GetSessionToken. Tokens could be created and used by attackers to move laterally and 
+escalate privileges.
+"""
+index = ["filebeat-*", "logs-aws*"]
```

Comment:
can you add `false_positives` and `references` please

Reply:
@brokensound77 yeah sorry. I was meaning to come back to this and finish it.